exercises/luhn/test/luhn_tests.erl

```diff
@@ -6,13 +6,21 @@
 
 
 invalid_test() ->
```

Comment:
Currently we have 2 tests here having multiple assertions each.

Can you please split them into multiple cases with a single assertion each?

name the function as the tests description (all lowercase, spaces replaced by underscores `_`) and `_test` appended. The latter is important, that `eunit` can recognize the test.

Also make sure they are in the same order as in the JSON.